doc/source/migrations/docs-changelog-setup.rst

```diff
@@ -257,15 +262,20 @@ Also, replace ``ansys.<product>.<library>`` with the name under ``tool.flit.modu
     showcontent = true
 
     [[tool.towncrier.type]]
-    directory = "dependencies"
-    name = "Dependencies"
+    directory = "maintenance"
+    name = "Maintenance"
     showcontent = true
 
     [[tool.towncrier.type]]
     directory = "miscellaneous"
     name = "Miscellaneous"
     showcontent = true
 
+    [[tool.towncrier.type]]
+    directory = "test"
+    name = "Test"
+    showcontent = true
+
```

Comment:
For conventional commits we should template all this section and inject it on runtime IMO. That way we prevent users having to define it on every single repo.